mappings/lowfat-macula-hebrew.xquery

```diff
@@ -284,7 +272,7 @@ declare function local:compound($nodeWithCChild)
 
 declare function local:phrase($node)
 {
-    if (local:oneword($node))
+    if (local:oneword($node)/m)
```

Comment:
This was throwing an error when I tried to process the `Node` on line 277 below unless I tested specifically for nodes with a child `m`.